settings.php

```diff
@@ -283,7 +291,7 @@
                                             </table>
                                         </div>
                                         <div class="form-group">
-                                            <textarea name="newuserlists" class="form-control" rows="1" placeholder="Enter one URL per line to add new blocklists"></textarea>
+                                            <input name="newuserlists" class="form-control" placeholder="Enter URL to add a new blocklists"></input>
```

Comment:
With the change in plurality, `blocklists` should be `blocklist`, and `a` should be inserted before `URL`